database/auth.go

```diff
@@ -52,6 +58,83 @@ func (db *Db) saveAccounts(paramsNumber int, accounts []types.Account) error {
 	return nil
 }
 
+// SaveVestingAccounts saves the given vesting accounts inside the database
+func (db *Db) SaveVestingAccounts(vestingAccounts []exported.VestingAccount) error {
+	if len(vestingAccounts) == 0 {
+		return nil
+	}
+
+	for _, account := range vestingAccounts {
+		switch vestingAccount := account.(type) {
+		case *vestingtypes.ContinuousVestingAccount, *vestingtypes.DelayedVestingAccount:
+			_, err := db.storeVestingAccount(account)
+			if err != nil {
+				return err
+			}
+
+		case *vestingtypes.PeriodicVestingAccount:
+			vestingAccountRowID, err := db.storeVestingAccount(account)
+			if err != nil {
+				return err
+			}
+			db.storeVestingPeriods(vestingAccountRowID, vestingAccount.VestingPeriods)
```

Comment:
We need to return the error here

```suggestion
			return db.storeVestingPeriods(vestingAccountRowID, vestingAccount.VestingPeriods)
```